core/src/main/java/greencity/service/UserService.java

```diff
@@ -298,6 +299,14 @@
     User updateUserProfilePicture(MultipartFile image, String email,
                                   UserProfilePictureDto userProfilePictureDto);
 
+    /**
+     * Delete user profile picture {@link User}.
+     *
+     * @param email {@link String} - email of user that need to update.
+     * @return {@link User}.
+     */
+    User deleteUserProfilePicture(String email);
```

Comment:
Can you please do not return `User`, in order not to expose `User` to controllers layer.

Reply:
Done